### PR TITLE
Gmf 2.5

### PR DESCRIPTION
**Pull request by kalbermattenm**
Add GMF 2.5 features & roadmap

@rbovard, could please review that ?

### CODE REVIEW

File: content/pages/4_roadmap.md [hunk 1]
```diff
@@ -9,28 +9,52 @@ Lang: en
...
 
-To be defined
+1. Improve WMS browser (search, server list)
+2. Map slider (swipe between maps)
+3. Improve redlining, be able to define length, size
+4. Close legend automatically when layer is deactivated
+5. Display snappable layers and enable activation/deactivation
+6. Intranet user panel
+7. Show and validate mandatory fields in editing panel
+8. Timeslider improvments (dynamic map rendering)
```
Comment (rbovard) [inline content/pages/4_roadmap.md]: improvments > improvements

File: content/pages/4_roadmap.md [hunk 2]
```diff
@@ -9,28 +9,52 @@ Lang: en
...
+9. Be able to define min and max scale for WMTS
+10. Let filters active even if panel is closed
+11. Query using a polygon
+12. Global loading & counter message
+13. Long name cut off in display window
+14. Story maps
+15. KML styling (import & export)
+16. Be able to use desktop on a tablet
+17. Geolocation on desktop interface
+18. WMS getFeatureInfo on non WFS servers
```
Comment (rbovard) [inline content/pages/4_roadmap.md]: getFeatureInfo > GetFeatureInfo

File: content/pages/4_roadmap.md [hunk 3]
```diff
@@ -9,28 +9,52 @@ Lang: en
...
 
-Version under development:
-
-* Functionalities status: In development
-* Development: Starting in Noember 2018
-* Estimated release date: 15 April 2019
-* EOS: 15 April 2022
+* Functionalities status: Frozen
+* Development: Done
+* Release date: 07 July 2019
```
Comment (rbovard) [inline content/pages/4_roadmap.md]: 07 > `7<sup>th</sup>`

File: content/pages/4_roadmap.md [hunk 4]
```diff
@@ -9,28 +9,52 @@ Lang: en
...
+* EOS: 07 July 2022
```
Comment (rbovard) [inline content/pages/4_roadmap.md]: 07 > `7<sup>th</sup>`

File: content/pages/4_roadmap_fr.md [hunk 5]
```diff
@@ -9,26 +9,52 @@ Lang: fr
...
+* Date estimée de sortie: fin 2019
 
 ### Fonctionnalités
 
-A définir
+1. Amélioration du navigateur WMS (recherche et liste de serveurs)
+2. Map slider (glisser entre deux cartes)
+3. Amélioration du dessin, pouvoir définir des longueurs et tailles
+4. Fermer automatiquement la légende lorsqu'une couche est désactivée
+5. Illustrer les couches accrochables et permettre l'activation / désactivation
```
Comment (rbovard) [inline content/pages/4_roadmap_fr.md]: Illustrer > Afficher

File: content/pages/4_roadmap.md [hunk 6]
```diff
@@ -9,28 +9,52 @@ Lang: en
...
 ### Functionalities
 
-To be defined
+1. Improve WMS browser (search, server list)
+2. Map slider (swipe between maps)
+3. Improve redlining, be able to define length, size
+4. Close legend automatically when layer is deactivated
+5. Display snappable layers and enable activation/deactivation
+6. Intranet user panel
+7. Show and validate mandatory fields in editing panel
```
Comment (rbovard) [inline content/pages/4_roadmap.md]: panel > form ?

File: content/pages/4_roadmap_fr.md [hunk 7]
```diff
@@ -9,26 +9,52 @@ Lang: fr
...
+6. Panneau de gestion Intranet
+7. Afficher et valider les champs obligatoires dans le panneau d'édition
```
Comment (rbovard) [inline content/pages/4_roadmap_fr.md]: panneau > formulaire

File: content/pages/4_roadmap_fr.md [hunk 8]
```diff
@@ -9,26 +9,52 @@ Lang: fr
...
+8. Améliorer le Timeslider (rendu dynamique de la carte)
```
Comment (rbovard) [inline content/pages/4_roadmap_fr.md]: Timeslider > slider temporel / curseur temporel ?

File: content/pages/4_roadmap_fr.md [hunk 9]
```diff
@@ -9,26 +9,52 @@ Lang: fr
...
+9. Echelles min et max pour les couches WMTS
+10. Laisser les filtres actifs quand le panneau est fermé
+11. Requête en utilisant un polygone
```
Comment (rbovard) [inline content/pages/4_roadmap_fr.md:38]: Requête > Sélection ?

File: content/pages/4_roadmap_fr.md [hunk 10]
```diff
@@ -9,26 +9,52 @@ Lang: fr
...
+12. Indicateur de chargement et compteur d'objet
```
Comment (rbovard) [inline content/pages/4_roadmap_fr.md:39]: objet > objets

File: content/pages/4_roadmap_fr.md [hunk 11]
```diff
@@ -9,26 +9,52 @@ Lang: fr
...
+13. Raccourcir les noms trop longs dans la fenêtre de résultats
+14. Story maps
+15. Style des KML (à l'importation & exportation)
```
Comment (rbovard) [inline content/pages/4_roadmap_fr.md]: à l'importation & exportation > import & export

File: content/pages/4_roadmap_fr.md [hunk 12]
```diff
@@ -9,26 +9,52 @@ Lang: fr
...
+16. Adapter le desktop pour une utilisation sur tablette
```
Comment (rbovard) [inline content/pages/4_roadmap_fr.md]: le desktop > l'interface desktop

File: content/pages/4_roadmap_fr.md [hunk 13]
```diff
@@ -9,26 +9,52 @@ Lang: fr
...
+17. Géolocalisation sur l'interface desktop
+18. WMS getFeatureInfo sur des serveurs ne supportant pas le WFS
```
Comment (rbovard) [inline content/pages/4_roadmap_fr.md]: getFeatureInfo > GetFeatureInfo

File: content/pages/4_roadmap_de.md [hunk 14]
```diff
@@ -9,24 +9,52 @@ Lang: de
...
+9. Definition von min und max Kartenmassstab von WMTS Layers
+10. Filter aktiv lassen selbst wenn Formular geschlossen wird
+11. Abfrage mit Polygon
+12. Globales Laden und Zähleranzeige
+13. Verkürzung langer Namen im Anzeigefenster
+14. Story maps
+15. KML Styling (Import & Export)
+16. Benützen der Desktopversion auf einem Tablet
+17. Geolocation auf der Desktopversion
+18. WMS getFeatureInfo auf nicht WFS Servern
```
Comment (rbovard) [inline content/pages/4_roadmap_de.md]: getFeatureInfo > GetFeatureInfo

File: content/pages/4_roadmap_fr.md [hunk 15]
```diff
@@ -9,26 +9,52 @@ Lang: fr
...
+19. Securité - Pas d'accréditation en cache
```
Comment (rbovard) [inline content/pages/4_roadmap_fr.md]: Securité > Sécurité

File: content/pages/4_roadmap_fr.md [hunk 16]
```diff
@@ -9,26 +9,52 @@ Lang: fr
...
+20. Securité - Blocage de compte configurable
```
Comment (rbovard) [inline content/pages/4_roadmap_fr.md]: Securité > Sécurité

File: content/pages/4_roadmap_fr.md [hunk 17]
```diff
@@ -9,26 +9,52 @@ Lang: fr
...
+21. Securité - Stockage sécurisé des mots de passe
```
Comment (rbovard) [inline content/pages/4_roadmap_fr.md]: Securité > Sécurité

File: content/pages/4_roadmap_fr.md [hunk 18]
```diff
@@ -9,26 +9,52 @@ Lang: fr
...
+22. Securité - Timeout des sessions
```
Comment (rbovard) [inline content/pages/4_roadmap_fr.md]: Securité > Sécurité

File: content/pages/4_roadmap_fr.md [hunk 19]
```diff
@@ -9,26 +9,52 @@ Lang: fr
...
+23. Architecture - Plus que Docker
```
Comment (rbovard) [inline content/pages/4_roadmap_fr.md]: Plus que Docker > Docker uniquement

**Review by rbovard — APPROVED** [review]
Thanks a lot @kalbermattenm!

I just noticed some typos otherwise LGTM :)

For the german version, maybe you can ask a review from someone of BS/BL?

**Comment by kalbermattenm** [comment]
Thanks a lot for the thoughtful review !

I have written the German page with @voisardf (thanks a lot by the way....)

**Comment by kalbermattenm** [comment]
I am merging this and will update the website shortly after.

**Pull request merged**